NPE in 3.1.0-SNAPSHOT when no qualifier is present

When building a simple pom without a qualifier with goal released-version, then the plugin crashes with a null pointer.

**Plugin declaration:**
```
<plugin>
  <groupId>org.codehaus.mojo</groupId>
  <artifactId>build-helper-maven-plugin</artifactId>
  <version>3.1.0-SNAPSHOT</version>
  <executions>
    <execution>
      <id>released-version</id>
      <goals>
        <goal>released-version</goal>
      </goals>
    </execution>
  </executions>
</plugin>
```

**Exception:**
```
Caused by: java.lang.NullPointerException
    at java.util.concurrent.ConcurrentHashMap.putVal (ConcurrentHashMap.java:1011)
    at java.util.concurrent.ConcurrentHashMap.put (ConcurrentHashMap.java:1006)
    at java.util.Properties.put (Properties.java:1316)
    at org.codehaus.mojo.buildhelper.AbstractDefinePropertyMojo.defineProperty (AbstractDefinePropertyMojo.java:47)
    at org.codehaus.mojo.buildhelper.ReleasedVersionMojo.defineVersionProperty (ReleasedVersionMojo.java:83)
    at org.codehaus.mojo.buildhelper.ReleasedVersionMojo.execute (ReleasedVersionMojo.java:128)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:208)
    ...
```

--- a/src/main/java/org/codehaus/mojo/buildhelper/ReleasedVersionMojo.java
+++ b/src/main/java/org/codehaus/mojo/buildhelper/ReleasedVersionMojo.java
@@ -25,6 +25,7 @@ package org.codehaus.mojo.buildhelper;
  */
 
 import java.util.List;
+import java.util.Objects;
 
 import org.apache.maven.artifact.ArtifactUtils;
 import org.apache.maven.artifact.factory.ArtifactFactory;
@@ -80,7 +81,7 @@ public class ReleasedVersionMojo
 
     private void defineVersionProperty( String name, String value )
     {
-        defineProperty( propertyPrefix + '.' + name, value );
+        defineProperty( propertyPrefix + '.' + name, Objects.toString( value, "" ) );
     }
 
     private void defineVersionProperty( String name, int value )

